tests/test_edge_cases.py

```diff
@@ -1795,22 +1819,38 @@ class Model(BaseModel):
     with pytest.raises(ValidationError) as exc_info:
         Model()
     assert exc_info.value.errors() == [
-        {'loc': ('nullable1',), 'msg': 'field required', 'type': 'value_error.missing'},
-        {'loc': ('nullable2',), 'msg': 'field required', 'type': 'value_error.missing'},
+        {'input': {}, 'loc': ('optional1',), 'msg': 'Field required', 'type': 'missing'},
+        {'input': {}, 'loc': ('nullable1',), 'msg': 'Field required', 'type': 'missing'},
+        {'input': {}, 'loc': ('nullable2',), 'msg': 'Field required', 'type': 'missing'},
```

Comment:
maybe not for this PR, but is `optional2` relying on inference of field types from default arguments? If we're getting rid of that we'll need to edit this test to something like:

````
    class Model(BaseModel):
        optional1: Any
        optional2: Optional[Any] = ...
        nullable1: Any = ...
        nullable2: Any = Field(...)
```

Reply:
good catch. I'm thinking based on this it might make sense to change it to:
```python
    class Model(BaseModel):
        optional1: Any
        optional2: Any = None
        optional3: Optional[Any] = None
        nullable1: Any = ...
        nullable2: Any = Field(...)
        nullable3: Optional[Any] = ...
```

Reply:
That sounds good to me